core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -156,6 +157,8 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
                     nativeConfig.enableAllSecurityServices |= Boolean.parseBoolean(prop.getValue());
                 } else if (prop.getKey().equals("quarkus.native.enable-all-charsets") && prop.getValue() != null) {
                     nativeConfig.addAllCharsets |= Boolean.parseBoolean(prop.getValue());
+                } else if (prop.getKey().equals("quarkus.native.enable-all-timezones") && prop.getValue() != null) {
+                    enableAllTimeZones = Boolean.parseBoolean(prop.getValue());
```

Comment:
I don't think removing the configuration property here is the solution.

Let's tackle the issue now. I would rename `addAllCharsets` to `enableAllCharsets` in the `nativeConfig` and maybe deprecate the old property and still somehow support it.

Reply:
I see your point and I agree. I'd have preferred to do it in another PR, since it is mostly unrelated :-)

Reply:
> I don't think removing the configuration property here is the solution.
> 
> Let's tackle the issue now. I would rename `addAllCharsets` to `enableAllCharsets` in the `nativeConfig` and maybe deprecate the old property and still somehow support it.

I'll create an issue for this.